test_word_search.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module contains unit tests for word search solution
+"""
+from word_search import get_valid_words, generate_grid, search_word
+
+FILE_NAME = 'words_dict.txt'
+WORDS = get_valid_words(FILE_NAME)
+GRID_SIZE = 15
```

Comment:
Is there a reason you made this a global? Are there other ways you could parameterize the test for grid size?

Reply:
I made WORDS global because it is used in a few test functions. But you are right, FILE_NAME and GRID_SIZE could be local.

Reply:
I'm not sure it would be reasonable to create this fixture. We also can make test grid_size parameterized